styleguide/template.md

```diff
@@ -129,20 +131,30 @@ When linking to an API, you can use its unique identifier (UID) that is auto-gen
 
 You can use one of the following syntax:
 
-1. Markdown link: `[link_text](xref:UID)`
-2. Auto link: `<xref:UID>`
-3. Shorthand form: `@UID`
+1. Auto-link: `<xref:UID>` or `<xref:UID?displayProperty=fullName>`
+
+ The displayProperty query parameter produces a fully qualified link text. Default behavior is to produce a link text that shows only the member or type name.
+ 
+2. Markdown link: `[link_text](xref:UID)`
+
+ Use only when the auto-link behavior is not the desired display. 
 
-- Example: `@System.String`
-- Example: `[String class](xref:System.String)` 
+Examples:
+- `<xref:System.String>` renders as [String](https://docs.microsoft.com/dotnet/api/system.string)
+- `<xref:System.String?displayProperty=fullName>` renders as [System.String](https://docs.microsoft.com/dotnet/api/system.string)
+- `[String class](xref:System.String)` renders as [String class](https://docs.microsoft.com/dotnet/api/system.string)
```

Comment:
Perhaps also add how to include member parameters.

Reply:
See earlier comment. I think an example using generics would also be very helpful.

Reply:
My point here was just to show how the rendering is formatted differently depending on the format you choose, so I chose a simple example. Otherwise, I need to introduce the encoding notes earlier for them to understand the example.
I like @rpetrusha's idea of including parameters. However, I'd avoid adding that now since the link behavior for them is broken in my opinion (It should be class.member and not namespace.class.member). I have a bug/feature filed already.